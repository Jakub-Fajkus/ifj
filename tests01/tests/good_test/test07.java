class Main {
	static void run(){
		double a = ifj16.readDouble();
		double b = ifj16.readDouble();
		double c = a + b;
		ifj16.print(a + " + " + b + " = " + c + "\n");
	}
}
