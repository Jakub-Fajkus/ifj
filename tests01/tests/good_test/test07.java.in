12.3
56.2
