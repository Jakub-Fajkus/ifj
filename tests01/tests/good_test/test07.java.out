12.3 + 56.2 = 68.5
